Sources/Widgets/Representations/ArrowHandleRepresentation/index.js

```diff
@@ -222,6 +217,7 @@ function vtkArrowHandleRepresentation(publicAPI, model) {
    *  - a first rotation to be oriented along model.rotation
    *  - an optional second rotation to face the camera
    * @param {vec3} scale3 Scale of the glyph, rotated when glyph is rotated.
+   * @returns {vec3} scale3 the vector
```

Comment:
@returns does not work exactly the same as for @Kitware/paraviewweb 
`* @returns {vec3} The input scale vector oriented`